pkg/webui/lib/errors/utils.js

```diff
@@ -118,9 +118,6 @@ export const isPermissionDeniedError = error => (
  */
 export const isUnauthenticatedError = error => (
   grpcStatusCode(error) === 16
```

Comment:
Should we have this as well as in `404` error?
```suggestion
  grpcStatusCode(error) === 16 || httpStatusCode(error) === 401
```

Reply:
We could do that, but then we should do the same for `isInternalError`, `isAlreadyExistsError` and `isPermissionDeniedError`.

Reply:
Done.